src/js/languages/default.json

```diff
@@ -66,6 +67,9 @@
     "pauseButton": "pause",
     "playButton": "play",
     "volumeDown": "volume down",
-    "volumeUp": "volume up"
+    "volumeUp": "volume up",
+    "description": "video audio description",
```

Comment:
Should we call this key `audioDescription` or `videoDescription` to tie in with the accessibility language?